### PR TITLE
Refs #10616 - use new pulp child ssl fragment for gpg key proxy

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
Requires https://github.com/Katello/puppet-pulp/pull/66

**Comment by jlsherrill** [comment]
[test]

**Comment by ehelms** [comment]
What do you think about a test that ensures this gets called with this content file if $pulp is true?

**Comment by jlsherrill** [comment]
@ehelms added test, not sure how to test the content, as it is a rendered template as a param value, any thoughts?

File: spec/classes/capsule_spec.rb [hunk 1]
```diff
@@ -45,6 +45,8 @@
 
     it { should contain_class('crane').with( {'key' => '/etc/pki/katello/private/katello-apache.key',
                                               'cert' => '/etc/pki/katello/certs/katello-apache.crt'} ) }
+
+    it { should contain_pulp__child__fragment('gpg_key_proxy') }
```
Comment (ehelms) [inline spec/classes/capsule_spec.rb]: I think you can do `with` here:

```
it 'should contain pulp gpg proxy' do
  should contain_pulp_child_fragment('gpg_key_proxy').with({
    :ssl_content => template('capsule/_pulp_child_gpg_proxy.erb'
  })
end
```

**Comment by jlsherrill** [comment]
@ehelms updated

**Comment by ehelms** [comment]
ACK

**Pull request merged**